src/runtime_src/xdp/profile/plugin/aie_debug/aie_debug_plugin.cpp

```diff
@@ -284,21 +284,34 @@ namespace xdp {
       std::stringstream msg;
       msg << "Unable to successfully execute AIE Profile polling kernel. " << e.what() << std::endl;
       xrt_core::message::send(severity_level::warning, "XRT", msg.str());
+      return;
     }
 
     XAie_ClearTransaction(&aieDevInst);
 
-    auto instrbo_map = instr_bo.map<uint8_t*>();
-    instr_bo.sync(XCL_BO_SYNC_BO_FROM_DEVICE);
-    
-    // TODO: figure out where the 8 comes from
-    instrbo_map += sizeof(XAie_TxnHeader) + sizeof(XAie_CustomOpHdr) + 8;
-    auto output = reinterpret_cast<aie_profile_op_t*>(instrbo_map);
+    static constexpr uint32_t SIZE_4K   = 0x1000;
+    static constexpr uint32_t OFFSET_3K = 0x0C00;
```

Comment:
lower case variable names, please.  Else you run the risk of preprocessor interfering when some obscure code out of your control defines these relatively common names.